circle.yml

```diff
@@ -15,3 +19,7 @@ test:
     - curl --retry 10 --retry-delay 2 -v http://localhost:5000
     # Run selenium tests
     - nosetests
+
+  post:
+    - killall sc
+    - while ! grep -q 'Goodbye' $SAUCE_LOG; do sleep 1; done  # wait until Sauce Connect closes the tunnel
```

Comment:
Maybe add a ```timeout: 10``` so it doesn't go on forever (10min) in the event the log message changes or something else